prices_voice.go

```diff
@@ -48,26 +52,31 @@ type VoicePrice struct {
 }
 
 type VoiceNumberPrice struct {
-	Country           string            `json:"country"`
-	IsoCountry        string            `json:"iso_country"`
-	Number            string            `json:"number"`
-	InboundCallPrice  InboundPrice      `json:"inbound_call_price"`
-	OutboundCallPrice OutboundCallPrice `json:"outbound_call_price"`
-	PriceUnit         string            `json:"price_unit"`
-	URL               string            `json:"url"`
+	Country            string              `json:"country"`
+	IsoCountry         string              `json:"iso_country"`
+	DestinationNumber  string              `json:"destination_number"`
+	OriginationNumber  string              `json:"origination_number"`
+	InboundCallPrice   InboundPrice        `json:"inbound_call_price"`
+	OutboundCallPrices []OutboundCallPrice `json:"outbound_call_prices"`
+	PriceUnit          string              `json:"price_unit"`
+
+	URL string `json:"url"`
 }
 
+// https://www.twilio.com/docs/voice/pricing#pricing-voice-country-instance-resource
 // returns the call price by country
 func (cvps *CountryVoicePriceService) Get(ctx context.Context, isoCountry string) (*VoicePrice, error) {
 	voicePrice := new(VoicePrice)
 	err := cvps.client.GetResource(ctx, voicePathPart+"/Countries", isoCountry, voicePrice)
 	return voicePrice, err
 }
 
-// returns the call price by number
-func (nvps *NumberVoicePriceService) Get(ctx context.Context, number string) (*VoiceNumberPrice, error) {
+// https://www.twilio.com/docs/voice/pricing#list-uri-by-origination-number
+// returns the call price by number or numbers pair
+func (nvps *NumberVoicePriceService) Get(ctx context.Context, destinationNumber string, data url.Values) (*VoiceNumberPrice, error) {
 	voiceNumPrice := new(VoiceNumberPrice)
-	err := nvps.client.GetResource(ctx, voicePathPart+"/Numbers", number, voiceNumPrice)
+	pathPart := strings.Join([]string{voicePathPart, "Numbers", destinationNumber}, "/")
+	err := nvps.client.MakeRequest(ctx, http.MethodGet, pathPart, data, voiceNumPrice)
```

Comment:
ah, you can use ListResource here I think

Reply:
I don't think so
There is not listable resource, I think new ParametrizedGet method will be more obviously.

anyway, done